met-web/src/components/engagement/form/EngagementWidgets/WhoIsListening/WhoIsListeningForm.tsx

```diff
@@ -90,6 +111,25 @@ const WhoIsListeningForm = () => {
                 <WidgetTitle widget={widget} />
                 <Divider sx={{ marginTop: '0.5em' }} />
             </Grid>
+            <Grid item xs={12}>
+                <MetLabel>Description (Optional)</MetLabel>
+                <TextField
+                    name="description"
+                    variant="outlined"
+                    value={listeningWidget?.description}
+                    onChange={(e) => {
+                        setListeningWidget({ ...listeningWidget, description: e.target.value });
+                    }}
+                    label=" "
+                    aria-label="Description: optional."
```

Comment:
No need to change things now, but in the future I'd advise that we use the actual label. If you have multiple elements that label a control, you can use `aria-labelledby="id1 id2"` where the IDs correspond to each labelling element. From WCAG: "Whenever possible, use the label element to associate text with form elements explicitly."

https://www.w3.org/WAI/tutorials/forms/labels/